docs/cli/oc4ids.rst

```diff
@@ -0,0 +1,123 @@
+OC4IDS Commands
+================
+
+The following commands may be used when working with `Open Contracting for Infrastructure <https://standard.open-contracting.org/infrastructure>`__ data.
```

Comment:
-> "when working with data conforming to the Open Contracting for Infrastructure Data Standards (OC4IDS)."